spring/src/main/java/io/cucumber/spring/api/CucumberTestContext.java

```diff
@@ -0,0 +1,10 @@
+package io.cucumber.spring.api;
+
+public final class CucumberTestContext {
+    public static final String SCOPE_CUCUMBER_GLUE = "cucumber-glue";
+
+    // Suppress default constructor for noninstantiability
+    private CucumberTestContext() {
+        throw new AssertionError();
```

Comment:
The exception is a bit excessive. If some one wants to use violence to get an instance of the `CucumberTestContext` then the consequences are on them.

Just an empty constructor will do. The comment isn't needed either. The pattern is obvious (e.g. sonar won't complain).